save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -228,7 +228,7 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
         when (state.selectedMenu) {
             OrganizationMenuBar.INFO -> renderInfo()
             OrganizationMenuBar.TOOLS -> renderTools()
-            OrganizationMenuBar.TESTS -> renderTests()
+            OrganizationMenuBar.BENCHMARKS -> renderTests()
```

Comment:
To my mind `renderTests` should also be renamed

Reply:
Fixed

Reply:
see no changes

Reply:
Saw in https://github.com/saveourtool/save-cloud/pull/2314/commits/46dfdbd5134e5ce6417e0d90cd776ca898dd1aa8